src/support/slack/commands/run-scrape.js

```diff
@@ -85,18 +85,23 @@ function RunScrapeCommand(context) {
       const urls = topPages.map((page) => ({ url: page.getURL() }));
       await say(`:white_check_mark: Found top pages for site \`${baseURL}\`, total ${topPages.length} pages.`);
 
-      const jobId = site.getId();
-      await triggerScraperRun(
-        jobId,
-        urls,
-        slackContext,
-        context,
-      );
-      await say(`:adobe-run: Triggered scrape run for site \`${baseURL}\` — Total ${urls.length} URLs`);
-
-      const message = `:white_check_mark: Completed triggering scrape runs for site \`${baseURL}\` — Total URLs: ${urls.length}`;
-
-      await say(message);
+      const half = Math.ceil(urls.length / 2);
```

Comment:
For handling a single URL: Does `triggerScraperRun` handle an empty URLs array gracefully?